capstone/src/main/webapp/restaurantSingUp.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>STEP Capstone</title>
+    <script src="signUp.js"></script>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+  </head>
+  <body>
+    <h1>Sign Up as a Restaurant!</h1>
+    <div id="sign-up">
+      <p>Name:</p>
+      <textarea id="name"></textarea>
```

Comment:
I might consider making the textarea ids a bit more specific. Maybe new-restaurant-name, new-restaurant-phone, etc?

Reply:
Done!